src/Commands/SetGearRoller.h

```diff
@@ -0,0 +1,19 @@
+#ifndef SetGearRoller_H
+#define SetGearRoller_H
+
+#include "Commands/Command.h"
+
+class SetGearRoller : public Command {
+public:
+	SetGearRoller(double rollerSpeedRPM);
+	void Initialize();
+	void Execute();
+	bool IsFinished();
+	void End();
+	void Interrupted();
+
+private:
+	double rollerSpeedRPM;
```

Comment:
wouldn't it be better to write this as `double mRollerSpeedRPM;`? If not both your constructor variable and class variable have the same name.